components/TabBar/TabBar.tsx

```diff
@@ -20,47 +30,47 @@ export const TabBar = ({ state, descriptors, navigation }: BottomTabBarProps) =>
           options.tabBarLabel !== undefined
             ? options.tabBarLabel.toString()
             : options.title !== undefined
-              ? options.title
-              : route.name
+            ? options.title
+            : route.name
```

Comment:
Ternary operators can be made a lot more readable using modern javascript:
```
options.tabBarLabel?.toString() ?? options.title ?? route.name
```

should do the same!

Reply:
I will address this thanks